mypy/test/data.py

```diff
@@ -556,6 +556,9 @@ def repr_failure(self, excinfo: Any) -> str:
 
 
 class DataSuite:
+    def __init__(self, *, update_data: bool = False) -> None:
```

Comment:
Since you're adding this here you should update all the classes that derive from it. Or you can follow the existing pattern and leave it out here but add such a `__init__` method to `ParserSuite` (like all other subclasses of `DataSuite` do). It seems these classes all need an `update_date` keyword argument because of the call on line 533 in data.py (line number from master, it's in `runtest()` in `MypyDataCase`).

Note that these classes don't seem to be instantiated anywhere, I suppose they are recognized dynamically. Do you know how?